mongo/mongo.go

```diff
@@ -104,7 +104,7 @@ func New(url, database string) (*MongoStorage, error) {
 	ms.authentications = client.Database(database).Collection("authentications")
 	ms.notifications = client.Database(database).Collection("notifications")
 	ms.userAccessProfiles = client.Database(database).Collection("userAccessProfiles")
-	ms.communitites = client.Database(database).Collection("communities")
+	ms.communities = client.Database(database).Collection("communities")
```

Comment:
This breaks everything @lucasmenendez 
We should think about how to push this fix...

Reply:
Because you are changing the collection name, and this has direct impact to the database. The collection will be empty.

Reply:
But only changes the name of the struct attribute, the collection name remains the same.

Reply:
The collection always had the correct `communities` name, it's just the struct property.